Tella/Scenes/CommonConnectionReport/ViewModels/OutboxMainViewModel.swift

```diff
@@ -0,0 +1,168 @@
+//
+//  OutboxMainViewModel.swift
+//  Tella
+//
+//  Created by gus valbuena on 7/10/24.
+//  Copyright © 2024 HORIZONTAL. All rights reserved.
+//
+
+import Foundation
+import Combine
+
+class OutboxMainViewModel<T: ServerProtocol>: ObservableObject {
+    
+    var mainAppModel : MainAppModel
+    var reportsViewModel : ReportMainViewModel
+    
```

Comment:
Is this viewModel of multiple reports or only one report? If it is only one maybe better to rename it to `reportMainViewModel`

Reply:
it contains all the reports of the connection. It manages the fetching of all the drafts, outbox and submitted reports and the navigation between the different pages

Reply:
Then it is better to rename the class to  `ReportsMainViewModel` to indicate that the view model handles all the reports